feature-libs/product-multi-dimensional/selector/components/guards/product-multi-dimensional-selector.guard.spec.ts

```diff
@@ -65,7 +65,7 @@ class MockSemanticPathService implements Partial<SemanticPathService> {
   }
 }
 
-describe('ProductMultiDimensionalGuard', () => {
+xdescribe('ProductMultiDimensionalGuard', () => {
```

Comment:
Is this intentional `xdescribe`?

Reply:
I've merged the main branch, there is no `xdescribe` anymore